src/en/wizden-staff/maintainer/review-procedure.md

```diff
@@ -1,54 +1,181 @@
-    - Any PR that carries the `Changes: Sprites` label needs to be approved by an Art Lead. This takes the form of the `S: Art Approval` label or an approval via Discord or Forums. Art Leads are responsible for ensuring the `S: Art Approval` label is applied to PRs when they approve of them.
-  - For closing, this takes the form of 2 Maintainers expressing an opinion to close the PR, or a reluctance to implement the PR, in the PR's comments. This "Disapproval" can also be given in written form during discussions on other platforms such as Discord or the SS14 forums, as long as it is also stated by another Maintainer in a comment under the PR.
-  - In the case where there's dissent among maintainers, e.g. 2 Approvals and 1 Disapproval, a Maintainer Discussion should be held as described under the Policy section.
-  - You do not need to be one of the two Maintainers who have given Approval/Disapproval to merge/close the PR. Ideally a PR should be processed once it meets the requirements to be merged/closed, but if that does not happen for some reason, any other Maintainer is free to do it (even if that Maintainer is the PR author).
-- PRs that have more than 3 weeks of inactivity waiting on something from the PR author should be considered "stale" and labeled as such with the `S: Stale` label, unless another reason explains the inactivity (e.g. freeze or author has informed about a longer break). You may use any means to contact the author of the PR, but at the minimum you should leave a comment on the PR and give at least 1 week to respond before closing a stale PR. A PR that is waiting on an action from Maintainers (e.g. a code review or discussion resolution) is not eligible to be marked as stale.
-- PRs that change public APIs, move code into another namespace, change a prototype ID / prototype definition in C# etc. should contain a "breaking changes" section. This section should contain a summary of the changes and how to apply them. Upon merging the PR it is the maintainers task to post the breaking changes section (Not just the PR link!) into `#codebase-changes`. The message format should first include a link to the PR, then the "breaking changes" section copied from the PR and lastly a ping for `@contrib-notifications.`. While not necessary, checking the message for spelling errors and correcting them is preferred.
+Any portion of this procedure may be waived and/or modified with permission from a Project Manager or Wizard.
```

Comment:
This should define where the permission can be given, probably Discord or GitHub comment?. Something like
`(...) may be waived and/or modified with *written* permission from (...)`